### PR TITLE
Adding Python examples for authentication with Key credential

### PR DESCRIPTION
**Pull request by C0x41lch0x41**

### CODE REVIEW

File: python/login.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        self.challengeIdentifier = ''
+        self.challenge = ''
+        self.challengeB64 = ''
+        self.authToken = ''
+        self.credential = None
+        self.firstFactor = {}
+        
+    def _setChallenge(self, challenge):
+        self.challenge = challenge
+        self.challengeB64 = base64.urlsafe_b64encode(challenge.encode('utf-8')).decode('utf-8').rstrip('=')
```
Comment (Thibault2ss) [inline python/login.py]: This should not be needed anymore, you can take the challenge as it is without encoding it

File: python/register.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        self.tempAuthToken = ''
+        self.challenge = ''
+        self.challengeB64 = ''
+        self.credential = None
+        self.firstFactorCredential = {}
+        
+    def _setChallenge(self, challenge):
+        self.challenge = challenge
+        # rstrip is needed so that challenge is matched by the backend
+        self.challengeB64 = base64.urlsafe_b64encode(challenge.encode('utf-8')).decode('utf-8').rstrip('=')
```
Comment (Thibault2ss) [inline python/register.py]: same here

File: python/login.py (hunk 1 above)
Comment (C0x41lch0x41) [reply python/login.py]: Yes I tested it, it is working. I removed the base64 part. Thanks.

**Comment by C0x41lch0x41** [comment]
@Thibault2ss are you ok to merge this?

**Review by Thibault2ss — APPROVED** [review]

**Review by ncornwell — APPROVED** [review]

**Pull request merged**